CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing
+
+- [Contributing](#contributing)
+  - [GitHub Flow](#github-flow)
+    - [How to work with GitHub Flow](#how-to-work-with-github-flow)
+  - [Working with Forks and Pull Requests](#working-with-forks-and-pull-requests)
+  - [Working with Issues / How to Contribute in Issues](#working-with-issues--how-to-contribute-in-issues)
+  - [How to Write Commit Messages](#how-to-write-commit-messages)
+  - [Code Review](#code-review)
+  - [Code of Conduct](#code-of-conduct)
+  - [Code Style](#code-style)
+  - [Contributor License Agreement (CLA)](#contributor-license-agreement-cla)
+
+## GitHub Flow
+
+For more information see [GitHub Flow](https://githubflow.github.io/)
+
+GitHub Flow is very lightweight (especially compared to GitFlow).
+This workflow uses only two kinds of branches:
+
+- Feature branch
+- Main branch (previously called master)
+
+The `feature` branches are used to develop new features as well as fixes.
```

Comment:
IMHO you should emphasize named things with `_` like _feature_, because the backticks are for preformatted text like code.